File: 0081_Search in Rotated Sorted Array II/solution.cpp
class Solution {
public:
    bool search(vector<int>& nums, int target) {
        int left = 0;
        int right = nums.size() - 1;
        while(left <= right) {
            if (nums[left] != target && nums[right] != target) {
                left++;
                right--;
            }
            else {
                return true;
            }
        }
        return false;
    }
};

File: 0081_Search in Rotated Sorted Array II/solution.py
class Solution:
    def search(self, nums: List[int], target: int) -> bool:
        left = 0
        right = len(nums) - 1
        while left <= right:
            if nums[left] != target and nums[right] != target:
                left += 1
                right += -1
            else :
                return True
        return False